docs/prow/obligatory-security-measures.md

```diff
@@ -4,7 +4,7 @@ Read about the obligatory security measures to take on a regular basis and when
 
 ## Change Secrets regularly
 
-All Secret used in the Prow production cluster must be changed every six months. Follow [Prow secret management](./prow-secrets-management.md) to create a new key ring and new Secrets. Then, use [Secrets populator](./../../development/tools/cmd/secretspopulator/README.md) to change all Secrets in the Prow cluster.
+All Secret used in the Prow production cluster must be changed every six months. Follow [Prow secret management](./prow-secrets-management.md) to create a new key ring and new Secrets. Then, [External Secrets Syncer](../../prow/workload-cluster/README.md) will change all Secrets in the Prow cluster automatically.
```

Comment:
```suggestion
All Secrets used in the Prow production cluster must be changed every six months. Follow the [Prow secret management](./prow-secrets-management.md) guidelines to create a new key ring and new Secrets. Once the key ring and new Secrets are ready, the [External Secrets Syncer](../../prow/workload-cluster/README.md) changes all Secrets in the Prow cluster automatically.
```
The link ->  [External Secrets Syncer](../../prow/workload-cluster/README.md) doesn't point the External Secrets Syncer